python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -101,8 +101,11 @@ def _dbt_nodes_to_assets(
         out_name_to_node_info[node_name] = node_info
         internal_asset_deps[node_name] = asset_deps
 
+    # unique id to prevent name collisions between multiple dbt multi-assets
+    uid = hash(tuple(outs.keys())) % 10000
+
     @multi_asset(
-        name="dbt_project",
+        name=f"dbt_project_{uid}",
```

Comment:
This means that the op name might shift as more models get added to the dbt project, right? Would it make sense to base it off of some combination of the project name and the selection so that it's more stable?

Reply:
Also, thought that's unrelated to the goal of this PR, but could possible make sense to change while we're shaking things up: would it make sense to call it `run_dbt_project` instead of `dbt_project`? I actually provisionally made that change for my SDA presentation to make it easier to understand that the shared string across these asset definitions is an op:

<img width="525" alt="image" src="https://user-images.githubusercontent.com/654855/161872083-411cd8ee-1d36-4055-a91f-ec5cde11ba6a.png">